<launch>
    <!--slam控制需要的参数--> 
    <param name="Is_slam_start" type="int" value="0" />
    <param name="slam_origin_get_once" type="bool" value="false" />
    
    <node
        pkg="gnss_localizer"
        type="simple_localization_slam_gnss_node"
        name="gnss_localizer_node"
        output = "screen">

#nanbu    
      <!--  <param name="map_longitude" value="106.60201333" />
        <param name="map_latitude" value="29.52848067" />
	    <param name="map_yaw" value="0" /> -->
#st slam_test
 	    <!-- <param name="map_longitude" value="106.60208666" />
        <param name="map_latitude" value="29.53769333" />
	    <param name="map_yaw" value="0" />  -->
#xue xiao
        <param name="map_longitude" value="106.601925836" />
        <param name="map_latitude" value="29.5377395072" />
        <param name="map_yaw" value="0" />

        <param name="is_display_trajectory" value="true" /> #是否显示轨迹
        <param name="is_save_trajectory" value="true" />    #是否保存轨迹，需要启用显示轨迹,默认存在/$SEED_HOME/data/navidation/trajectory.txt
    </node>
</launch>


